platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/ConfigurationFileProxy.java

```diff
@@ -69,46 +75,29 @@ public ConfigurationFileProxy(ConfigurationStore configurationStore) {
         }
     }
 
-    protected BundleContext getContext() {
-        Bundle cxfBundle = FrameworkUtil.getBundle(ConfigurationFileProxy.class);
-        if (cxfBundle != null) {
-            return cxfBundle.getBundleContext();
-        }
-        return null;
+    public void writeSolrConfiguration(String core) {
```

Comment:
The core is not known until a Solr factory is called to create a core.  This method is only called by the Solr factories.  I can make it package private.